includes/apple-exporter/components/class-pre.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Apple_Exporter\Components;
+
+/**
+ * An HTML's blockquote representation.
+ *
+ * @since 1.1.8
+ */
+class Pre extends Component {
+
+  /**
+   * Look for node matches for this component.
+   *
+   * @param DomNode $node
+   * @return mixed
+   * @static
+   * @access public
+   */
+  public static function node_matches( $node ) {
+    if ( 'pre' == $node->nodeName ) {
```

Comment:
Consistency with other similar classes in the plugin. It's not in scope for this pull request to update them all.